cypress/test/search-results.spec.ts

```diff
@@ -0,0 +1,34 @@
+import SearchResultsSteps from '../steps/search-results-steps';
+
+describe('Onto Search Results Component', () => {
+  it('Should find the component', () => {
+    SearchResultsSteps.visit();
+    SearchResultsSteps.getOntoSearchResultsComponent();
```

Comment:
I know that cypress will blow if it's not there, but better add `.should('be.visible')` at least